community-plugins.json

```
@@ -4668,5 +4668,13 @@
         "description": "Create folder of snippets to activate them in one click !",
         "repo": "Mara-Li/obsidian-group-snippets",
         "branch": "master"
+    },
+    {
+        "id": "obsidian-markdown-file-suffix",
+        "name": "More Markdown file suffix (.mdx/.svx).",
```

```suggestion
        "name": "MDX/SVX file support",
```

Maybe something like this is better?